R/pkg/R/DataFrame.R

```diff
@@ -2814,7 +2815,7 @@ setMethod("except",
 #' path <- "path/to/file.json"
 #' df <- read.json(path)
 #' write.df(df, "myfile", "parquet", "overwrite")
-#' saveDF(df, parquetPath2, "parquet", mode = saveMode, mergeSchema = mergeSchema)
+#' saveDF(df, parquetPath2, "parquet", mode = "overwrite")
```

Comment:
the example with these parameter are actually intentional - it is to show what additional properties can be set.
but I'd agree if they run as-is it would be preferred